### DISCUSSION TITLE
Pull Requests as A Teaching Tool | /en/2017/09/pull-requests-as-a-teaching-tool/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#962] [Comments]
https://yihui.org/en/2017/09/pull-requests-as-a-teaching-tool/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
<https://github.com/rbind/yihui/blob/663c8b49ed8a65dc9d36cd10c0827d350e52f2be/static/css/custom.css#L18-L28>

> Originally posted on 2017-09-18 22:01:28

**Comment by chuxinyuan** [comment]
不知道为什么，我用鼠标选中某一段的时候，很容易把这一段以上的段落也选中，除非鼠标离段首字母比较近时，箭头变成 “I” 的时候再向后拖动才能选中当前段落。

**Reply by yihui** [reply]
我的鼠标没有这个问题啊。是只有我的网站能复现这个问题吗？

**Reply by chuxinyuan** [reply]
Manjaro Linux，Firefox 117.01 下你的网站有这个问题，但是我的网站是正常的。然后我又去 Win10 Edge 浏览器试了下，没这个问题了，很奇怪。

**Reply by chuxinyuan** [reply]
Win10 下 Firefox 浏览器打开你的网站也存在这个问题。

**Reply by yihui** [reply]
我在苹果电脑上相同版本的火狐浏览器也无法复现这个问题，鼠标选哪个段落就是哪个，不会自动扩展到相邻段落。我暂时没有别的操作系统可测试，你可以看看别人能否复现。

**Reply by hlblade** [reply]
是的，我的电脑上firefox也有这个问题。

**Reply by yihui** [reply]
谢谢，我无法复现的问题还真是不好解决。中文页面也有同样的问题吗？比如：https://yihui.org/cn/about/

**Reply by chuxinyuan** [reply]
是的，中文页面也有同样的问题。好在用火狐浏览器的应该不多。

**Reply by yihui** [reply]
那可真是怪了。待我抽空寻一台 Windows 电脑试试。

**Comment by chuxinyuan** [comment]
鼠标放在标题上面时，标题后面的 "#" 感觉没有显示的必要。

**Reply by yihui** [reply]
那是为了给用户一个机会复制锚点链接，这样的链接分享出去之后，别人点击了就可以直接跳转到指定的节。

**Reply by chuxinyuan** [reply]
细节拉满，赞！

**Comment by hlblade** [comment]
About没有这个问题发自我的手机
